client/src/annoMatrix/whereCache.ts

```diff
@@ -49,27 +49,38 @@ creates a cache entry of:
 }
 */
 import { _getColumnDimensionNames } from "./schema";
-import { _hashStringValues } from "./query";
+import { _hashStringValues, Query } from "./query";
+import { Field, Schema } from "../common/types/entities";
+
+export interface WhereCache {
+  summarize?: {
+    [key: string]: {
+      [key: string]: WhereCacheTerms;
+    };
+  };
+  where?: {
+    [key: string]: WhereCacheTerms;
+  };
+}
+
+interface WhereCacheTerms {
+  [key: string]: Map<string, Map<string, string[] | number[]>>;
```

Comment:
I have typed column label here as `string[] | number[]` but I suspect this should actually just be `number[]`.

Reply:
Hmm yeah it seems to be `Int32Array` by looking at what `client/src/util/dataframe/labelIndex.ts:47` returns, but the test file `client/__tests__/util/annoMatrix/whereCache.test.ts` has examples using `string[]`. Maybe it supports both?

Reply:
I see, interesting. So it looks like we have:

*  `Int32Array` in `client/src/util/dataframe/labelIndex.ts:47`
* `string[]` in `client/__tests__/util/annoMatrix/whereCache.test.ts:19`
* `number[]` in `client/__tests__/util/annoMatrix/whereCache.test.ts:102`

Should I update the union type to include all three (`string[] | number[] | Int32Array`) or do you think the tests might need updating to match the app functionality (i.e. just `Int32Array`)?

Reply:
You want all three, ie, `string[] | number[] | Int32Array`

Reply:
Added/updated [whereCache.ts::WhereCacheColumnLabels](https://github.com/chanzuckerberg/cellxgene/blob/mim/annoMatrix/client/src/annoMatrix/whereCache.ts#L66).

Reply:
Looks good, with the caveat that `WhereCacheColumnLabels` is slightly incorrect.  See inline comment I left on the type definition.

Reply:
Updated [WhereCacheColumnLabels](https://github.com/chanzuckerberg/cellxgene/blob/mim/annoMatrix/client/src/annoMatrix/whereCache.ts#L67).